PurchasesCoreSwift/Logging/Logger.swift

```diff
@@ -21,42 +28,45 @@ import Foundation
     }
 }
 
-@objc(RCLog) public class Logger: NSObject {
-    @objc public static var logLevel: LogLevel = .info
-    @objc public static var logHandler: (LogLevel, String) -> Void = { level, message in
+class Logger {
```

Comment:
could be a `struct` instead? 🤔

Reply:
In this case I think controlling identity here with a class makes sense. Also, pass-by-value for a Logger feels wrong to me.